src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -198,7 +199,10 @@ public Task Stop()
                             await ProduceEnd();
 
                             // Finish reading the request body in case the app did not.
-                            await RequestBody.CopyToAsync(Stream.Null);
+                            while (await RequestBody.ReadAsync(_nullBuffer, 0, _nullBuffer.Length) != 0)
+                            {
+                                ;
```

Comment:
Might add comment instead? I find empty loop bodies jarring - like the loop is there by mistake